api/python/cellxgene_census/src/cellxgene_census/experimental/ml/geneformer_tokenizer.py

```diff
@@ -0,0 +1,145 @@
+import pickle
+from typing import Any, Dict, Optional, Sequence, Set
+
+import numpy as np
+import numpy.typing as npt
+import scipy
+import tiledbsoma
+
+from .cell_dataset_builder import CellDatasetBuilder
+
+try:
+    import geneformer
+except ImportError:
+    # pyproject.toml can't express Geneformer git+https dependency
+    raise ImportError(
+        "Please install Geneformer with: " "pip install git+https://huggingface.co/ctheodoris/Geneformer@39ab62e"
+    ) from None
```

Comment:
@pablo-gar Here's how I've handled the Geneformer dependency for now. As we discussed, there are a couple problems with making it a conventional Python dependency (pyproject.toml doesn't yet allow this `git+https` scheme; and one of its transitive dependencies needs cython). And it's just the two .pkl files I'm using from it.

Reply:
Just pasting my comment from slack to record it here on just copying the files into the repo 

> Yes I’ve thought about this. I don’t have any concerns from the license perspective. My concern is more related to future iterations of Geneformer, specifically when they train again and their files change. I think is less error prone for the end-user to have the dependency taking care of the versioning.
> I do understand the hurdles of adding a dependency on Geneformer, but my thought is … if you (user) are going to create a Dataset to be used in Geneformer why would you do that if you are not going to use it with Geneformer?